projects/packages/changelogger/src/AddCommand.php

```diff
@@ -266,10 +266,16 @@ protected function execute( InputInterface $input, OutputInterface $output ) {
 			// Determine the changelog entry and add to the file contents.
 			$entry = $input->getOption( 'entry' );
 			if ( $isInteractive ) {
+				// Additional formatting info if we're adding the changelog to a plugin.
+				if ( false !== strpos( getcwd(), '/projects/plugins/' ) ) {
+					$formatNeeded = "Please use the format 'Feature: Description' e.g. 'Stats: Fixes funny errors.'\n";
+				} else {
+					$formatNeeded = '';
+				}
```

Comment:
We've kept the changelogger package fairly unopinionated until now, and avoided adding use-cases that would only be useful or apply to the Jetpack monorepo. I'm a bit wary of adding something like this in now. I wonder if there would be a way to edit that wording without adding that additional check in the package itself.

@anomiex Do you have any recommendations there?

Maybe we should limit our recommendation to the CLI tool for now?

Reply:
That's a good point. The problem with just adding it to the CLI is if someone runs `jetpack changelog add`, it still just calls the changelog script. The prompts defined in the CLI only run in cases where we detect a changelog file is needed in multiple projects, so just adding it there likely won't be seen enough to have an effect.

We could shelve this iteration unless we think of something more specific we can implement for Jetpack itself.

Reply:
Yeah, let's not tie Changelogger to the monorepo. If we want to be able to do warnings like this or whatever, it should be configuration or we should add more plugin functionality so the monorepo can plug in.